acis_thermal_check/tests/acisfp/acisfp_test_spec.json

```diff
@@ -441,6 +441,17 @@
             794
         ]
     },
+    "limits": {
+        "fptemp": {
+            "planning.data_quality.high.acisi": -112.0,
+            "planning.data_quality.high.aciss": -111.0,
+            "planning.data_quality.high.aciss_hot": -109.0,
+            "planning.data_quality.high.cold_ecs": -118.2,
```

Comment:
Noting here that the actual *data* limit for cold ECS data is -117.2 C.

Reply:
@jazan12 that's right, but in order to avoid hitting that limit, we plan to -118.2 C, correct?